tests/as_python_test.py

```diff
@@ -10,6 +10,10 @@ class dummy_class:
     def __init__(self):
         self.i = 0
 
+def gen_tree(depth):
+    if depth == 0: return SExp.to(1337)
+    return SExp.to((gen_tree(depth - 1), gen_tree(depth - 1)))
```

Comment:
```suggestion
    subtree = gen_tree(depth-1)
    return SExp.to((subtree, subtree))
```
I think this will be `O(n)` instead of `O(2**n)`.